### PR TITLE
Rename Sufia::FilesControllerBehavior to Sufia::FileSetsControllerBehavior

### LINKED ISSUE #1379: Should Sufia::FilesControllerBehavior be Sufia::FileSetsControllerBehavior
_From @grosscol on November 12, 2015 17:18_

It looks like the functionality there is being mixed into the FileSetsController.  Is this naming a hold over from the GenericFile history?

_Copied from original issue: projecthydra-labs/sufia-core#187_

### PR DESCRIPTION
**Pull request by awead**
fixes #1379

### CODE REVIEW

File: app/controllers/concerns/sufia/file_sets_controller_behavior.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ module FilesController
     autoload :LocalIngestBehavior
```
Comment (mjgiarlo) [inline app/controllers/concerns/sufia/file_sets_controller_behavior.rb]: @awead I see that on line 2 we're still referencing a `FilesController` and we have some directory structure around this to pull in the other modules. Do you think this PR should expand to nuke any and all references to "FilesController" (or "files_controller") in the code, the filenames, and directory names? Or shall we merge this PR and ticket that as a follow-on?

**Comment by mjgiarlo** [comment]
:+1: (though see my :question:)

File: app/controllers/concerns/sufia/file_sets_controller_behavior.rb (hunk 1 above)
Comment (awead) [inline app/controllers/concerns/sufia/file_sets_controller_behavior.rb]: @mjgiarlo yeah, I wondered about that too... this also prompted my question in #1424. I would go with a follow up ticket and perhaps more discussion on the issue above.

Comment (mjgiarlo) [inline app/controllers/concerns/sufia/file_sets_controller_behavior.rb]: :ok_hand:

**Pull request merged**